abstractions/php/src/RequestInfo.php

```diff
@@ -0,0 +1,93 @@
+<?php
+namespace Microsoft\Kiota\Abstractions;
```

Comment:
FYI we're renaming request info to request information see #559. I know it's a bit late in the game but Nikitha caught a pretty important issue.